content/blog/2018-12-28-subquery.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Subquery
+created_at: 2018-12-28
+kind: article
+author_name: Ganesh Vernekar
+---
+
+## Introduction
+
+As the title suggests, subquery is a part of a query, and allows you to do a range query within a query, which was not possible before. It has been a long standing request on Prometheus [prometheus/prometheus/1227](https://github.com/prometheus/prometheus/issues/1227) and a desired feature.
+
+PR for subquery was recently merged into Prometheus [prometheus/prometheus/4831](https://github.com/prometheus/prometheus/pull/4831), and will be available in Prometheus 2.7 release. Let’s learn more about it below.
```

Comment:
PR -> The PR

Better: spell out "PR" -> "The pull request".

"for subquery" -> "for subquery support"

No comma before "and will be available".

"in Prometheus 2.7 release" -> "in the Prometheus 2.7 release" (or "in Prometheus 2.7").